ngx_http_sxg_filter_module.c

```diff
@@ -266,17 +273,40 @@ static ngx_int_t subresource_fetch_handler(ngx_http_request_t* req, void* data,
   if (req->done) {
     return NGX_OK;
   }
+  if (req->upstream->headers_in.status_n != 200) {
+    // Even if fetching subresource failed, we ignore it.
+    --ctx->subresources;
+    return NGX_OK;
+  }
+
   if (req->out->buf->last - req->out->buf->pos == req->upstream->length) {
     ngx_http_set_ctx(req, ctx, ngx_http_sxg_filter_module);
     sxg_buffer_t integrity = sxg_empty_buffer();
     sxg_buffer_t new_header_entry = sxg_empty_buffer();
 
-    if (calc_integrity(req, &integrity) &&
-        sxg_write_string("<https://", &new_header_entry) &&
+    // Searche 'as' statement of original link header.
+    ngx_str_t as = ngx_null_string;
+    ngx_array_t* subresource_list = &ctx->subresource_list;
+    ngx_subresource_t* subresource = subresource_list->elts;
+    for (int i = 0; i < subresource_list->nelts; ++i) {
```

Comment:
It looks like this is `O(nelts^2)` given [the outer for loop](https://github.com/google/nginx-sxg-module/blob/1da319d87b99ab84653d4668018b22aac5e8808c/ngx_http_sxg_filter_module.c#L454). Am I reading that right? If so, this is probably fine because it's expected to be small (<20?), and the contents of this loop are fast, but maybe we should leave a TODO or open a bug.